region/norfair/west/Ice Beam Gate Room.json

```diff
@@ -1069,18 +1069,23 @@
         "h_artificialMorphMovement",
         {"or": [
           {"and": [
-            "canTrickyJump",
-            "h_artificialMorphSpringBall",
-            "h_artificialMorphPowerBomb"
+            "h_artificialMorphPowerBomb",
+            "h_additionalBomb"
+          ]},
+          {"and": [
+            "h_artificialMorphPowerBomb",
+            "canInsaneJump"
           ]},
+          "canTrickyGMode",
           {"enemyDamage": {"enemy": "Sm. Dessgeega", "type": "contact", "hits": 2}}
         ]},
         "h_artificialMorphPowerBomb"
       ],
       "clearsObstacles": ["A", "B", "C"],
       "flashSuitChecked": true,
       "note": [
-        "It is tricky but possible to cross the room without any damage by jumping over the Small Dessgeega on room entry then using a Power Bomb to kill several enemies.",
+        "It is tricky but possible to cross the room without any damage by utilizing Samus' i-frames on room entry.",
+        "jumping over the Small Dessgeega on room entry then using a Power Bomb to kill several enemies.",
```

Comment:
Being able to jump isn't the default state for the strat

Reply:
Wow, whoops. I have no idea where that line came from. Probably a copy/paste that was half edited, haha.